internal/cli/templates.go

```diff
@@ -153,31 +153,21 @@ func resourceUsageTemplate() string {
 
 Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
 `,
-		ansi.Faint("Usage:"),
```

Comment:
We're making these headings bold so they are more easily displayed for the user.